st2common/st2common/util/pack.py

```diff
@@ -102,6 +104,13 @@ def validate_config_against_schema(config_schema, config_object, config_path,
 
     pack_name = pack_name or 'unknown'
 
+    for key in config_object:
+        if (jinja_utils.is_jinja_expression(value=config_object.get(key)) and
+                "decrypt_kv" in config_object.get(key) and config_schema.get(key).get('secret')):
+            raise ValueValidationException('Validation Error: decrypt_kv jinja filter'
+                                           ' specified for auto decrypted fields marked'
+                                           ' with `secret: True`')
```

Comment:
Please make the error message more friendly.  I had a hard time understanding what error this is. I think you can skip the `Validation Error:` prefix in the message.

Reply:
I second that :+1:
Would be also useful if error message will say what action user should take to fix the issue.

Reply:
Done